validator/sawtooth_validator/networking/handlers.py

```diff
@@ -212,10 +212,10 @@ def handle(self, connection_id, message_content):
         if connection_id in self._last_message:
             if time.time() - self._last_message[connection_id] < \
                     self._allowed_frequency:
-                LOGGER.debug("Too many Pings in %s seconds before "
-                             "authorization is complete: %s",
-                             self._allowed_frequency,
-                             connection_id)
+                LOGGER.warning("Too many Pings in %s seconds before "
```

Comment:
Could you add the time difference here?  It would be helpful to know frequently node is sending the ping.